### ISSUE TITLE
[ Master Schedule ] Show not displaying the correct end time when set

### ISSUE THREAD

**Issue by thomasmarshall1998** [netmix/radio-station#490]
Hi,

Have a techy issue I have tried to schedule a 3 hour show every Saturday "Dance Party Weekly with Lee Everest" should be showing as 10pm till 1pm which I have scheduled it for this time but on my schedule its showing as ending at 2am not 1am? not sure why..  I have the timing set to WordPress time all my other shows are fine except for this one 
![Screenshot 2024-03-29 164753](https://github.com/netmix/radio-station/assets/165409817/2f93a415-e6ee-49e7-9e8e-926fef3792f0)

**Comment by thomasmarshall1998** [comment]
![Screenshot 2024-03-29 164927](https://github.com/netmix/radio-station/assets/165409817/ce38febf-ae8f-4582-9999-25d8dfcb9b6a)

**Comment by tonyzeoli** [comment]
@thomasmarshall1998 Good catch on this possible bug. This is a show that spans two specific days, Saturday into Sunday.. Maybe there is an issue in the code that does not recognize Saturday night into Sunday morning while the other days have no issue. Not sure just yet. For now, I would set two consecutive Show times. One for 10 pm to 12 midnight and the other for 12 midnight to 1 am on the next day. I've tasked this to @majick777 for a look.

**Comment by tonyzeoli** [comment]
@majick777 This hasn't been looked at. Can you take a glance and see what the issue is here?
